nameable_person.rb

```diff
@@ -0,0 +1,6 @@
+class NameablePerson < Person
+  include Nameable
+  def correct_name
+    @name
+  end
+end
```

Comment:
- [ ] @jkanyi-web Great work adding the nameable module, however according to project requirement you are supposed to have a nameable class that has a method called correct_name that will raise a NotImplementedError, also this method doesn't inherit from the person class. Kindly consider removing the nameable module.
![image](https://github.com/jkanyi-web/OOP_School_Library/assets/61408860/a1c67eb9-539b-43f8-9c70-a139cab410d6)